### DISCUSSION TITLE
Bring back sv behavior on taiko for converts in classic mod

### DISCUSSION THREAD

**Discussion by akmal1223233** [ppy/osu#18670] [Ideas] [answered]
i find that sv in taiko converts are absent (even with classic mod on)

**Comment by bdach** [comment, marked as answer]
please specify examples of maps that you find problematic, ideally with timestamps

**Reply by akmal1223233** [reply]
https://osu.ppy.sh/beatmapsets/1434388#taiko/3242466 (just an example)

**Reply by smoogipoo** [reply]
Right, makes sense why it broke: https://github.com/ppy/osu/issues/18672

**Comment by smoogipoo** [comment]
They never went away. Works fine for me (tested on https://osu.ppy.sh/beatmapsets/65994).
